### ISSUE TITLE
Uncaught exception getLeft() 

### ISSUE THREAD

**Issue by michalsrutek** [PSD-Company/duo-navigation-drawer#20]
Hi,

I'm dealing with the following problem:
I start an Activity from which I instantiate a fragment that contains DuoDrawerLayout. From within that fragment I try to call another fragment, but I get 

> UncaughtException: java.lang.NullPointerException: Attempt to invoke virtual method 'int android.view.View.getLeft()' on a null object reference
>         at nl.psdcompany.duonavigationdrawer.views.DuoDrawerLayout.computeScroll(DuoDrawerLayout.java:284)
....

I am pretty sure that I do this after everything has been initialized. Any ideas?

**Comment by Alexander-1504** [comment]
It sounds like your content view is empty. Maybe this happens when you switch fragments?

**Comment by michalsrutek** [comment]
Yes, I'm immediately switing to a other fragment, but I believe only after everything has been initialized. Is there a way to programatically check that? Or maybe a callback for when the DuoDrawerLayout is successfully initialized?

**Comment by michalsrutek** [comment]
I am still not sure what is the cause... But I was able to get a workaround by calling
`menuDrawer.getContentView();` in the fragment from which I want to immediately switch to another one. `menuDrawer` is `DuoDrawerLayout`. And by the way, I'm using AndroidAnnotations 4.4.0 with this.
